src/lazy.js

```diff
@@ -349,7 +349,7 @@ export default function (Vue) {
       })
       freeList.forEach(item => {
         remove(this.ListenerQueue, item)
-        item.destroy()
+        item.$destroy()
```

Comment:
I suppose it must be ompatible with `destroy` and `$destroy`

```
item.destroy && item.destroy()
item.$destroy && item.$destroy()
```
or change reactiveListener.destroy to $destroy ?

Reply:
I think in this case this would be enough (and simple):

```
const destroy = item.destroy || item.$destroy;
destroy();
```

But unifying method names would be better. So we could do in `lazy-component`:

```
methods: {
  destroy () {
    return this.$destroy;
  }
}
```

Reply:
I also think that it is better to unify method names.

Reply:
Hi @hilongjw 

Are those changes fine with you?

This bug is killing my Sentry.io quota ;)